### PR TITLE
Create E2E test for operator

### PR DESCRIPTION
**Pull request by arturobrzut**

### CODE REVIEW

File: common/scripts/tests/tests.bats [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+}
+
+@test "Remove CR from IBMLicensing" {
+  kubectl delete IBMLicensing --all
+  [ $? -eq 0 ]
+}
+
+@test "Wait 120s for checking pod in ibm-common-services. List should be empty" {
+  sleep 120
```
Comment (adamdyszy) [inline common/scripts/tests/tests.bats:95]: delete all sleeps and change them for k8s solution like:
```bash
kubectl rollout status --timeout=<number of seconds>s -w deployment/<dep_name> --namespace=<ns>
```

**Review by adamdyszy — CHANGES_REQUESTED** [review]

**Review by adamdyszy — APPROVED** [review]

File: common/scripts/tests/tests.bats (hunk 1 above)
Comment (arturobrzut) [reply common/scripts/tests/tests.bats]: ok

**Pull request merged**